compliance_checker/cf/cf.py

```diff
@@ -688,73 +657,73 @@ def check_units(self, ds):
         metadata_vars = self._find_metadata_vars(ds)
         ancillary_vars = self._find_ancillary_vars(ds)
 
-        for k, v in ds.variables.items():
+        for name, variable in ds.variables.items():
 
             # skip climatological vars, boundary vars
-            if v in clim_vars or \
-               v in boundary_vars or \
-               v in metadata_vars or \
-               v in ancillary_vars or \
-               k in container_vars:
+            if name in clim_vars or \
+               name in boundary_vars or \
+               name in metadata_vars or \
+               name in ancillary_vars or \
+               name in container_vars:
```

Comment:
have all of these methods been updated for the name/variable consistency thing?  boundary vars does not look it to me: https://github.com/lukecampbell/compliance-checker/blob/bf45f8bda048c680f0ac4c7ecaa644c529db6d8f/compliance_checker/cf/cf.py#L338-L358